python/perf-kernels/flash-attention.py

```diff
@@ -966,9 +967,70 @@ def backward(ctx, do, _):
 @pytest.mark.parametrize('causal', [False, True])
 @pytest.mark.parametrize('use_bias', [False])
 def test_op_fwd(Z, H, N_CTX_Q, N_CTX_K, D_HEAD, causal, use_bias, dtype=torch.float16):
-    # TODO: using bias causes coredump for certain configs and must be fixed.
+    torch.manual_seed(20)
+    sm_scale = D_HEAD ** -0.5
+    input_metadata = MetaData(sm_scale=sm_scale)
+    input_metadata.max_seqlens_q = N_CTX_Q
+    input_metadata.max_seqlens_k = N_CTX_K
+    if causal:
+        input_metadata.need_causal()
+    if use_bias:
+        bias = torch.randn((1, H, N_CTX_Q, N_CTX_K), dtype=torch.float32, device="cuda")
```

Comment:
May be remove bias from this one along with the parametrize decorator? It is covered by the test below right?

Reply:
done